src/components/avatar.tsx

```diff
@@ -0,0 +1,18 @@
+import { twJoin } from "tailwind-merge";
+
+interface AvatarProps {
+  imageUrl?: string;
+  className?: string;
+}
+
+export function Avatar(props: AvatarProps) {
+  const { imageUrl, className } = props;
```

Comment:
I tested passing an undefined value to imageUrl property and the result was an icon of crashed image. I suggest you create a better fallback to this situation.

Here's the screenshot:
![image](https://github.com/cristianoprogramador/FE_SH/assets/49873623/2bbfdd4d-59f4-496c-8859-930117941798)

Reply:
i fixed that by using a default svg picture for when its undefined, thanks for the heads up